### PR TITLE
Num class

### PR DESCRIPTION
**Pull request by rpmertz**

### CODE REVIEW

File: test/test_sym.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    
+    # entropy = (1000*entropy)//(1/1000)
+    print ("mode", mode, "entropy", entropy)
+
+    test_result = (mode=="a" and 1.37 <= entropy and entropy <=1.38)
+    print (test_result)
+    
+    assert test_result
+
+test ()
```
Comment (sreedhara-aneesh) [inline test/test_sym.py:30]: Don't call your `test()` method in the file, the test engine will call it itself.

File: test/test_the.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+#append the relative location you want to import from
+sys.path.append("../src")
+import sym as classs
+
+def test():
+    test_result = True
+    
+    assert test_result
+
+test ()
```
Comment (sreedhara-aneesh) [inline test/test_the.py:17]: Don't call your `test()` method in the file, the test engine will call it itself.

**Review by sreedhara-aneesh — CHANGES_REQUESTED** [review]
Don't call test methods within the test files, the test methods will be called by the test engine itself. Left comments where this needs to be addressed. While this is fine for now since all tests are passing anyways, if there are any failing tests it will mess up the test engine.

**Review by mkudith — APPROVED** [review]
looks good to me

**Review by sreedhara-aneesh — APPROVED** [review]

**Pull request merged**